recipes/cls_explain.yaml

```diff
@@ -0,0 +1,11 @@
+workflow_hooks:
+    - type: WFProfileHook
+      name: profile_hook
+
+stages:
+  - name: cls-explanation
+    type: ClsExplainer
+    task: classification
+    mode: explain
+    config: {{ fileDirname }}/stages/classification/class_incr.yaml
```

Comment:
Is it okay to use the configs from `class_incr.yaml` ? If `explainer` needs special configs, then how about using independent config file?